tests/server/test_app.py

```diff
@@ -125,6 +160,22 @@ def test_invalid_thread_pinning():
     with pytest.raises(ValueError, match='Expected one of {"core","numa","none"}.'):
         _build_app(
             ServerConfig(
-                num_cores=1, num_workers=1, engine_thread_pinning="asdf", endpoints=[]
+                num_cores=1,
+                num_workers=1,
+                engine_thread_pinning="asdf",
+                endpoints=[],
+                loggers=None,
             )
         )
+
+
+@mock_engine(rng_seed=0)
+def test_default_logging_server_launches(engine_mock):
```

Comment:
Can you move this test into the test_endpoints file? And then also use a `TestClient(_build_app(server_config))` to send requests instead of requests lib

Reply:
moving, can't use TestClient here because the prometheus server does not run on FastAPI (the request here isn't going to deepsparse)